WMF Framework/WMFCaptchaViewController.swift

```diff
@@ -4,6 +4,23 @@ import UIKit
 // Presently it is assumed this view controller will be used only as a
 // child view controller of another view controller.
 
+extension UIView {
+    fileprivate func wmf_subviews(hide: Bool) {
```

Comment:
It just would have clearer naming -  instead of `self.wmf_subviews(hide: true)` it could read more clearly as `self.wmf_areSubviewsHidden = true` or `self.wmf_hideSubviews(true)`